src/chainstate/burn/db/burndb.rs

```diff
@@ -1168,6 +1168,15 @@ impl BurnDB {
         }
         Ok(burn_total)
     }
+
+    pub fn get_block_winning_vtx_index(conn: &Connection, block_hash: &BurnchainHeaderHash) -> Result<Option<u16>, db_error> {
```

Comment:
If you can spare the 30 seconds to do this, can you change the name to `_vtxindex` from `_vtx_index` to keep the codebase consistent?

Reply:
Yep